scripts/build.py

```diff
@@ -217,13 +214,15 @@ def _minify(source_path, target_path):
     # out-of-memory error.
     # https://circleci.com/blog/how-to-handle-java-oom-errors/
     # Use relative path to avoid java command line parameter parse error on
-    # Windows.
+    # Windows. Convert to posix style path on Windows because the Windows
```

Comment:
This raises the question of why a syntax error would be caused here but not elsewhere. Perhaps say something like "because the filepath arguments to the Java command need to be in posixpath style"?

Reply:
Updated